spec.html

```diff
@@ -26367,14 +26461,16 @@ <h1>
 
             <emu-alg>
               1. Assert: _state_.[[IsLoading]] is *true*.
-              1. If _module_ is a Cyclic Module Record, _module_.[[Status]] is ~new~, and _state_.[[Visited]] does not contain _module_, then
+              1. If _module_ is a Cyclic Module Record, _module_.[[Status]] is ~new~, _state_.[[Visited]] does not contain _module_, and _loadType_ is ~recursive-load~, then
```

Comment:
My personal preference is to check for `_loadType_` first since it drives whether to recur at all.